app/src/main/java/util/room/daos/PlaceDao.kt

```diff
@@ -31,14 +31,20 @@ import com.guerinet.mymartlet.model.place.Place
 abstract class PlaceDao : BaseDao<Place>() {
 
     /**
-     * Returns all of the places as an observable
+     * Returns all of the [Place]s as an observable
      */
     @Query("SELECT * FROM Place")
     abstract fun getPlaces(): List<Place>
 
+    /**
+     * Returns all of the user's favorite [Place]s
+     */
     @Query("SELECT id FROM Place WHERE ${Category.FAVORITES} IN(categories)")
     abstract fun getFavoritePlaces(): List<Int>
 
+    /**
+     * Delete the [Place]s with the given [ids]
```

Comment:
Isn't it when the id isn't in `ids`?

`"DELETE FROM Place WHERE Place.id NOT IN(:ids)"`

Perhaps this is pruning the place list vs deleting the place list